lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb

```diff
@@ -1034,16 +1123,40 @@ def apply_efficiency_ecm_hs09_ccashp_baseboard(model)
               supply_fan = fan # assume supply fan has higher pressure drop
             end
           end
-          fan_power = supply_fan.autosizedMaximumFlowRate.to_f * max_pd / supply_fan.fanTotalEfficiency.to_f
+          # There is an error in EnergyPlus in the estimated capacity of the coil "CoilCoolingDXVariableSpeed".
```

Comment:
what is the EnergyPlus error?

Reply:
@mdahlhausen This error was confirmed with the EnergyPlus team and they actually have a ticket for it. The reported autosized capacity of this coil is not correct. In this code a work-around is implemented to correct the capacity till a a fix is implemented by the E+ team.

Reply:
got it thanks